service/metrics/metrics.go

```diff
@@ -0,0 +1,20 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	// FlavorsUsedCounter is a Prometheus metric, counting the number of clusters created per flavor
+	FlavorsUsedCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "infra",
+			Name:      "flavors_used",
+			Help:      "Number of cluster created by flavor",
```

Comment:
```suggestion
			Help:      "Number of clusters created by flavor",
```

sorry for not spotting earlier / providing wrong suggestion.